### PR TITLE
React Native Release 4.3.0

### PR DESCRIPTION
**Pull request by AnatolyPristensky**
* [x] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [ ] Are the files formatted correctly?
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Bumped the versions
Updated TestApp to use latest SDKs
Updated the CHANGELOG.md

### CODE REVIEW

**Review by DmitriyKirakosyan — DISMISSED** [review]

**Review by aleksandr-dorofeev — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,29 @@
 # App Center SDK for React Native Change Log
 
-## Version 4.2.1 (Under development)
+## Version 4.3.1 (Under development)
```
Comment (AnastasiaKubova) [inline CHANGELOG.md:3]: Not need in this PR

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,12 +2,28 @@
...
+
+## Version 4.3.0
+
+### App Center
+
+#### Android/iOS
+
+* **[Feature]** Improved `AES` token encryption algorithm using `Encrypt-then-MAC` data authentication approach.
+
+
```
Comment (AnastasiaKubova) [inline CHANGELOG.md:15]: please remove empty line

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,12 +2,28 @@
 
 ## Version 4.2.1 (Under development)
 
+___
+
```
Comment (AnastasiaKubova) [inline CHANGELOG.md:6]: please remove it

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,12 +2,28 @@
 
 ## Version 4.2.1 (Under development)
```
Comment (AnastasiaKubova) [inline CHANGELOG.md:3]: remove it

**Review by DmitriyKirakosyan — APPROVED** [review]

**Review by AnastasiaKubova — APPROVED** [review]

**Pull request merged**